database.php

```diff
@@ -0,0 +1,38 @@
+
+<?php
+// Step 1: Create a connection
+$db = new PDO('mysql:host=db; dbname=cocktails', 'root', 'password');
```

Comment:
This code would do really well in a function. That way you can create a DB connection on multiple pages without needing to copy this code on every page.